File: backend/src/main/java/com/sergey/hotels/dto/Review.java
package com.sergey.hotels.dto;

public class Review {
    private String userName;
    private int raiting;
    private boolean approved;

    protected Review() {}

    public Review(String userName, int raiting, boolean approved) {
        this.userName = userName;
        this.raiting = raiting;
        this.approved = approved;
    }

    public String getUserName() {
        return userName;
    }

    public int getRaiting() {
        return raiting;
    }

    public boolean isApproved() {
        return approved;
    }

    @Override
    public String toString() {
//        System.out.println(userName);
//        System.out.println(raiting);
//        System.out.println(approved);
//        return userName;
        String str = "{ userName: " + userName + ",\n" +
                     "raiting: " + raiting + ",\n" +
                     "approved: " + approved + " }";
        return str;
    }
}

File: frontend/src/actions/review.js
import {
  SEND_REVIEW_START,
  SEND_REVIEW_SUCCESS,
  SEND_REVIEW_ERROR,
} from '<constants>/review';


export const sendReviewStart = () => ({
  type: SEND_REVIEW_START,
});

export const sendReviewSuccess = payload => ({
  type: SEND_REVIEW_SUCCESS,
  payload,
});

export const sendReviewError = payload => ({
  type: SEND_REVIEW_ERROR,
  payload,
});

export const sendReview = () => (dispatch, getState) => {
  dispatch(sendReviewStart());

  const url = 'http://localhost:8080/hotels/review';
  const { hotels = {} } = getState();
  const {
    reviewUser,
    reviewRating,
    reviewApproved,
  } = hotels;

  const body = {
    userName: reviewUser,
    raiting: reviewRating,
    approved: reviewApproved,
  };
  console.log('body', body);

  fetch(url, {
    method: 'POST',
    credentials: 'same-origin',
    headers: {
      'Content-Type': 'application/json',
    },
    body: JSON.stringify(body),
  })
    .then(response => response.json())
    .then(items => dispatch(sendReviewSuccess(items)))
    .catch(err => dispatch(sendReviewError(err)));
};
